### DISCUSSION TITLE
Azure Notification Hubs

### DISCUSSION THREAD

**Discussion by antonioberetini** [ionic-team/capacitor#4471] [Q&A]
Is there a better way to integrate with Azure notification hubs or should we default to this Cordova integration  https://github.com/derek82511/cordova-azure-notification-hubs

@mlynch this is next on our plate to get integrated. I want to make sure that we are taking the most optimal path.

**Comment by dbrils** [comment]
I'm also interested in knowing the best way to integrate Push Notifications with Azure Notifications Hub. It's unfortunate that the Capacitor documentation only shows the [integration with Firebase](https://capacitorjs.com/docs/guides/push-notifications-firebase). Especially since the I feel like the `cordova-azure-notification-hubs` package is old (last updated 3 years ago). And we have ANH SDKs for [iOS](https://github.com/Azure/azure-notificationhubs-ios) and [Android](https://github.com/Azure/azure-notificationhubs-android). I can't find to seem any sample implementation for this.
